### ISSUE TITLE
FSNS Raw Images?

### ISSUE THREAD

**Issue by Shikib** [tensorflow/models#4187]
I apologize if this was answered somewhere else, I wasn't able to find anything pertaining to my question.

Is there a way of downloading the raw FSNS images/labels anywhere? I have an OCR model implemented in PyTorch that I'd like to train/evaluate with the FSNS dataset, however, it appears that the download links provided in this repo only download the data in the form of TFRecords.

If there is no raw dataset available, would it be possible to direct me to something that might allow me to parse out the TFRecords and construct a raw dataset?

Thanks in advance!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by robieta** [comment]
It looks like TFRecords is the official format. Tagging @alexgorban as this seems like a reasonable request.

**Comment by robieta** [comment]
I'm afraid I don't see this happening in the near future. Sorry I couldn't be of more help.
